config/locales/en/instance_user_role_requests.yml

```diff
@@ -0,0 +1,7 @@
+en:
+  instance_user_role_requests:
+    new:
+      header: 'Create Role Request'
```

Comment:
👍  Will change to `Request to be an instructor or administrator`, because there is a select box for them to choose. It's slightly longer but more clear